semantic-source/src/Source/Language.hs

```diff
@@ -32,7 +32,6 @@ data Language
     | JavaScript
     | JSON
     | JSX
-    | Markdown
```

Comment:
This is a breaking change to the `Enum` instance, but I don’t think this is a problem, as at API interface boundaries this translation is carried out on text values.